shoop_tests/utils/test_enum.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from shoop.core.models.products import ProductMode
+from shoop.utils.enums import map_enum
+
+
+def test_map_enum():
+    assert map_enum(ProductMode, 0) is ProductMode.NORMAL
```

Comment:
What's the point of mapping int to enum with this function? Doesn't it produce unreadable code (because of magic numbers)?